tests/stock_dashboard_api/views/test_stock_data_view.py

```diff
@@ -1,12 +1,20 @@
 import json
+import os
+
 from unittest.mock import patch
 
 from stock_dashboard_api import app
 from stock_dashboard_api.models.stock_data_models import StockData
+from stock_dashboard_api.views import stocks_data_view
+
+# Flask app configuration for testing
+app.config.from_object(os.environ.get('FLASK_TESTING_CONFIG'))
+app.register_blueprint(stocks_data_view.test_mod)
```

Comment:
Why do you need to register blueprint in test module?

Reply:
to test CRUD functionality that prohibited for use by the user